app/models/memory.rb

```diff
@@ -7,7 +7,10 @@ class Memory < ActiveRecord::Base
            through: :keyword_associations,
            source: :memories
 
-  has_attached_file :image, default_url: "/img-300-placeholder.gif"
+  has_attached_file :image, styles: {
+      large: "600x600>", medium: "450x450>"
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.